WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -222,9 +222,11 @@ class AztecPostViewController: UIViewController, PostEditor {
                                                         .font: Fonts.title,
                                                         .paragraphStyle: titleParagraphStyle]
 
-        let textView = UITextView()
+        let textView =
+            UIApplication.shared.isCreatingScreenshots() ? UITextViewWithoutCaret() : UITextView()
 
         textView.accessibilityLabel = NSLocalizedString("Title", comment: "Post title")
+        textView.accessibilityLabel = "aztec-editor-title"
```

Comment:
Is this meant to be `textView.accessibilityIdentifier` rather than `textView.accessibilityLabel`?

Reply:
Yikes – you're absolutely right. Nice catch!